shell/platform/android/io/flutter/embedding/android/FlutterView.java

```diff
@@ -607,6 +611,15 @@ public void attachToFlutterEngine(
     for (FlutterEngineAttachmentListener listener : flutterEngineAttachmentListeners) {
       listener.onFlutterEngineAttachedToFlutterView(flutterEngine);
     }
+
+    // If the first frame has already been rendered, notify all first frame listeners.
+    // Do this after all other initialization so that listeners don't inadvertently interact
+    // with a FlutterView that is only partially attached to a FlutterEngine.
+    if (didRenderFirstFrame) {
```

Comment:
just call onFirstFrameRendererListener.onFirstFrameRendered here?

Reply:
I'm not sure what you're suggesting, can you elaborate?

Reply:
<not super important, feel free to do what you think makes sense> I just meant could we avoid duplicating the logic here and just call onFirstFrameRenderListener.onFirstFrameRendered() which basically does the same thing here?

Reply:
Ah, I see what you're saying now. Yeah I think we can make that change.